.github/workflows/ci.yml

```diff
@@ -21,9 +21,9 @@ on:
   schedule:
     - cron: '28 0 * * *'
   push:
-    branches: ['master', 'v1-10-test', 'v1-10-stable']
+    branches: ['master', 'v1-10-test', 'v1-10-stable', 'v2-0-test']
```

Comment:
We should remove it from direct pushes until we diverge. Since we are going to rebase v2-0-test on master this will trigger unnecessary buld every time v2-0-test is rebased to master.